ipn/ipnlocal/tailfs.go

```diff
@@ -179,20 +179,20 @@ func (b *LocalBackend) tailfsRemoveShareLocked(name string) (map[string]string,
 	}
 	fs.SetShares(shares)
 
-	return shareNameMap(shares), nil
+	return copyShares(shares), nil
 }
 
-func shareNameMap(sharesByName map[string]*tailfs.Share) map[string]string {
-	sharesMap := make(map[string]string, len(sharesByName))
+func copyShares(sharesByName map[string]*tailfs.Share) map[string]*tailfs.Share {
```

Comment:
is this just https://pkg.go.dev/maps#Clone ?

But it converts nil maps to non-nil zero length maps?

Reply:
Oh hey, maps.Clone is nice, will use that.